homeassistant/components/enphase_envoy/config_flow.py

```diff
@@ -58,7 +57,6 @@ class EnphaseConfigFlow(ConfigFlow, domain=DOMAIN):
     VERSION = 1
 
     _reauth_entry: ConfigEntry
```

Comment:
I purposefully left out the changes to reauth_entry for a follow-up PR as I think this one is big enough and reauth interferes with zeroconf